scripts/lib/CIME/case/check_input_data.py

```diff
@@ -6,9 +6,45 @@
 from CIME.XML.inputdata import Inputdata
 import CIME.Servers
 
-import glob
+import glob, hashlib
 
 logger = logging.getLogger(__name__)
+# The inputdata_checksum.dat file will be read into this hash if it's available
+chksum_hash = dict()
+
+
+def _download_checksum_file(server, input_data_root):
+    """
+    Return True if successfully downloaded
+    """
+    rel_path = "../inputdata_checksum.dat"
+    full_path = os.path.join(input_data_root, "inputdata_checksum.dat")
+    protocol = type(server).__name__
+    if protocol == 'svn':
```

Comment:
It seems like this logic is specific to our current svn inputdata server, and should be encoded in the xml defining the different servers rather than generally skipping this for any svn server.

Reply:
you are correct, i've added the file name to the config_inputdata.xml